dotnet/samples/KernelSyntaxExamples/Example00_03_OpenApiSkill_PlayFab.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+using Microsoft.SemanticKernel.Skills.OpenAPI.Extensions;
+using Newtonsoft.Json;
+using RepoUtils;
+
+/// <summary>
+/// This example shows how to import PlayFab APIs as skills.
+/// </summary>
+// ReSharper disable once InconsistentNaming
+public static class Example_00_03_OpenApiSkill_PlayFab
```

Comment:
>Example_00_03_OpenApiSkill_PlayFab [](http://example.com/codeflow?start=20&length=34)

Cool. so this example shows how to invoke playfab api through the kernel. 

A great next step would be to make that (also) used in a more complex generative complex (example 00_02) where it can gets an existing segment and suggest modifying it OR suggest (and performs) the creation of a new segment